app.ts

```diff
@@ -0,0 +1,238 @@
+		//     content?: Request,
+		//     callback?: (response: Response) => void,
+		//     errorCallback?: (err: any) => void) {
+		// const request = new XMLHttpRequest();
+		// request.open(method, url, true);
+		// request.onload = function () {
+		//     if (this.status >= 200 && this.status < 400) {
+		//         // Success!
+		//         const data = JSON.parse(this.response) as Response;
+		//         callback && callback(data);
+		//     } else {
+		//         // We reached our target server, but it returned an error
+
+		//         console.log("Error 404");
+		//     }
+		// };
+		// }
+		//region Data Loading methods
+		async function LoadRecordsData(fromID: number, toID: number): Promise<number[]> {
+			try {
+				const recordsvalue = await getRecordsCall(fromID, toID)
+				let DisplayContent = '';
+				for (const record of recordsvalue) {
+					DisplayContent += `<tr id="table-row-${record[0]}">`;
+					for (const column of record) {
+						DisplayContent += `<td align="center">${column}</td>`;
+					}
+					DisplayContent += '</tr>';
+					$("#wrapper-table-content-body").empty();
+					$("#wrapper-table-content-body").append(DisplayContent);
+				}
+			} catch (error) {
+				console.log("Error <No data>" + error)
+			}
+			return [fromID, toID];
+		}
+		function RecordsFromCursor(cursor: number[]): Promise<number[]> {
+			cursor = cursor.sort((a, b) => { return a - b });
+			return LoadRecordsData(cursor[0], cursor[1]);
+			// throw new Error("Error");
+		}
+		window.onresize = () => {
+			try {
+				const nextToId = calculateToId(previous[0]);
+				clearTimeout(Load);
+				Load = setTimeout(async () => {
+					const recordCount = await getRecordCountCall();
+					if (nextToId >= recordCount - 1) {
+						const fromId = recordCount - 1 - (calculateToId(previous[0]) - previous[0]);
+						const toId = recordCount - 1;
+						previous = await LoadRecordsData(fromId, toId);
+					} else {
+						previous = await LoadRecordsData(previous[0], nextToId)
+					}
+				}, 10);
+			} catch (error) {
+				// throw new Error("Error" + error);
+			}
+		}
+		// Handlers
+		async function LoadPageContent(fromID: number, toID: number): Promise<number[]> {
+
+			try {
+				let DisplayContent = "";
+				const columns = await getColumnNamesCall();
+				for (const column of columns) {
+					DisplayContent += `<th align="center">${column}</th>`;
+					$("#wrapper-table-header-row").empty();
+					$("#wrapper-table-header-row").append(DisplayContent);
+				}
+			} catch (error) {
+				console.log("Error <No Data>" + error);
+			}
+			return await LoadRecordsData(fromID, toID);
+		}
+		function ConvertNumber(input: string | number, parseAsInt: boolean = true): number {
+			switch (typeof input) {
+				case ('string'):
+					if (parseAsInt == true) {
+						return parseInt(input as string);
+					}
+					return parseFloat(input as string);
+				case ("number"):
+					return input as number;
+				default:
+					return 0;
+			}
+			throw new Error("Error");
```

Comment:
Because of the default case in your switch statement this line will never hit. If you must have a return outside of your switch, then make it the same as your default.